.github/workflows/ubuntu_release.yml

```diff
@@ -0,0 +1,50 @@
+name: Ubuntu release
+
+on:
+  push:
+    tags:
+      - '*'
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    env:
+      CC: gcc
+    steps:
+    - uses: actions/checkout@v4
+
+    - name: Install stuff for creating packages
+      run: sudo apt install -y fakeroot rpm dpkg debhelper
+
+    - name: Install packages required for optional configurations of cntlm
+      run: sudo apt install -y libkrb5-dev
+
+    - name: Install further tools (clang-tools for scan-build)
+      run: sudo apt install -y clang-tools

```

Comment:
`clang-tools` are not needed for building packages

Reply:
Removed